src/extension/app/aem-sidekick.js

```diff
@@ -43,6 +43,15 @@ export class AEMSidekick extends LitElement {
     });
   }
 
+  /**
+   * Allow the location the sidekick is working with the be accessed via
+   * the aem-sidekick element.
+   * @returns {URL}
+   */
+  get location() {
```

Comment:
I believe rum tracking was actually broken with the new sidekick without this.

See
https://github.com/adobe/aem-sidekick/blob/main/src/extension/rum.js#L44